wasm_utils/src/api_serialization/mod.rs

```diff
@@ -0,0 +1,13 @@
+/// This module holds structs for all arguments and return types
+/// that get serialized and deserialized between core native and
```

Comment:
I guess it's true that this encapsulates stuff that gets serialized and deserialized, but we do also have the RibosomeErrorCode stuff which is another really important definition for the interface across wasm.